server/planning/planning/planning.py

```diff
@@ -1831,7 +1831,30 @@ def _iter_recurring_plannings_to_update(self, updates, original, update_method):
     "description_text": metadata_schema["description_text"],
     "internal_note": {"type": "string", "nullable": True},
     "anpa_category": metadata_schema["anpa_category"],
-    "subject": metadata_schema["subject"],
+    "subject": {
```

Comment:
would be good to import it from events schema and just copy it here (or vice versa)

Reply:
done